src/components/EventPagination/EventPagination.tsx

```diff
@@ -0,0 +1,38 @@
+import ReactPaginate from "react-paginate";
+import { IoIosArrowBack, IoIosArrowForward } from "react-icons/io";
+import { IEventPagination } from "@/components/EventPagination/EventPagination.d";
+
+const EventPagination = (props: IEventPagination) => {
+  const { currentPage, pageCount, onPageChange } = props;
+  const marginPages = pageCount > 4 ? 1 : 0;
```

Comment:
@farzanashampa7 apu can we put this 4 inside appConstants so that in the future if it needs to be changed it can be changed easily.